service/src/main/java/org/apache/griffin/core/metastore/hive/HiveMetaStoreProxy.java

```diff
@@ -55,6 +59,28 @@ Licensed to the Apache Software Foundation (ASF) under one
 
     private IMetaStoreClient client = null;
 
+    @Value("${hive.krb5conf.path}")
+    private String hiveKrb5confPath;
+
+    @Value("${hive.keytab.path}")
+    private String keytabPath;
+
+    @Value("${hive.keytab.user}")
+    private String keytabUser;
+
+    @Value("${hive.need.kerberos}")
+    private String needKerberos;
```

Comment:
Can be a boolean.

Reply:
emmm,last pr for this variable 'hive.need.kerberos' i also think it type is boolean.
but in  org.apache.griffin.core.metastore.hive.HiveMetaStoreServiceJdbcImpl line 73: needKerberos define as string.

Reply:
Agree with @chitralverma 

@lovelyqincai Could we  change `needKerberos` variable in `org.apache.griffin.core.metastore.hive.HiveMetaStoreServiceJdbcImpl` to boolean type in another PR.

Reply:
ok,i also agree with.

Reply:
@wankunde 

> Agree with @chitralverma
> 
> @lovelyqincai Could we change `needKerberos` variable in `org.apache.griffin.core.metastore.hive.HiveMetaStoreServiceJdbcImpl` to boolean type in another PR.

ok,i also agree with.